tx.go

```diff
@@ -250,7 +251,12 @@ func (tx *Tx) close() {
 	} else {
 		tx.db.removeTx(tx)
 	}
+
+	// Clear all references.
```

Comment:
The function doesn't have any other exit points. I'm not sure a `defer` would help.